common/azure-common-test/src/main/java/com/azure/common/test/utils/SdkContext.java

```diff
@@ -3,58 +3,151 @@
 
 package com.azure.common.test.utils;
 
-import com.azure.common.test.TestMode;
-import com.azure.common.test.models.RecordedData;
+import com.azure.common.http.HttpHeader;
+import com.azure.common.http.HttpResponse;
+import reactor.core.Exceptions;
+import reactor.core.publisher.Mono;
 
-import java.util.Objects;
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.net.URI;
+import java.nio.charset.StandardCharsets;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.zip.GZIPInputStream;
+
+import static com.azure.common.test.utils.TestConstant.DEFAULT_BUFFER_LENGTH;
 
 /**
  * The class to contain the common test methods for testing SDK.
  */
-public class SdkContext {
-    private final ResourceNamer nameGenerator;
-
-    /**
-     * Creates an SDK context that keeps track of variable names for the recorded data.
-     *
-     * <ul>
-     *     <li>If the {@code testMode} is {@link TestMode#PLAYBACK}, this will generate random variables by reading from
-     *     {@code recordedData}.</li>
-     *     <li>If the {@code testMode} is {@link TestMode#RECORD}, this will generate random variables and write them to
-     *     {@code recordedData}.</li>
-     * </ul>
-     *
-     * @param testMode The test for this context.
-     * @param recordedData The data to persist or read any variables names to or from.
-     */
-    public SdkContext(TestMode testMode, RecordedData recordedData) {
-        Objects.requireNonNull(recordedData);
-        nameGenerator =  new TestResourceNamer("", testMode, recordedData);
-    }
+public final class SdkContext {
```

Comment:
I don't know why this class is called SdkContext anymore. It seems like it is now just a TestUtils class or similar.